labs/api/codemonkey_endpoints.py

```diff
@@ -1,6 +1,6 @@
 from fastapi import APIRouter, HTTPException
 from labs.api.types import CallLLMRequest, CodeMonkeyRequest, GithubModel
-from labs.config import LITELLM_API_KEY
+from labs.config.settings import LITELLM_API_KEY
```

Comment:
the import for env variables must be `from labs.config import settings` and then use `settings.LITELLM_API_KEY`